active/0021-auto-subscribe.md

```diff
@@ -0,0 +1,86 @@
+# EMQX Auto Subscribe
+
+## Changelog
+
+* 2022-07-06: @DDD
+
+## Abstract
+
+With Auto Subscription enabled, after a client is successfully connected to EMQX, it may not need to send SUBSCRIBE requests to EMQX, because EMQX will complete the pre-defined subscriptions for the client.
+
+Prior to version 5, this feature is also known as "proxy subscription".
+
+## Motivation
+
```

Comment:
below is a wishing list from earlier versions which motivates this new design

Reply:
What I mean is that the configuration is not in array form and will determine which database the current configuration is based on the type.